FlipViewExtensions implementation (#387)
unoplatform/uno.toolkit.ui: doc/helpers/FlipView-extensions.md

# SelectorExtensions Attached Properties

Provides additional features for `FlipView`.

## Properties
Property|Type|Description
-|-|-

> Next|FlipView| Sets the `FlipView` to will be controlled.

Reviewer: ```suggestion
Next|FlipView| Sets the `FlipView` that should be moved to its next item
```

> Previous|FlipView| Sets the `FlipView` that will controled.

Reviewer: ```suggestion
Previous|FlipView| Sets the `FlipView` that should be moved to its previous item
```

## Usage

```xml
xmlns:utu="using:Uno.Toolkit.UI"
...

<Grid>
    <FlipView>
        <FlipView.Items>
            <Grid Background="Azure">
                <Button
                    HorizontalAlignment="Left"
                    utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
                    Content="Previous" />
                <Button
                    HorizontalAlignment="Right"
                    utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
                    Content="Next" />
            </Grid>
            <Grid Background="Blue">
                <Button
                    HorizontalAlignment="Left"
                    utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
                    Content="Previous" />
                <Button
                    HorizontalAlignment="Right"
                    utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
                    Content="Next" />
            </Grid>
        </FlipView.Items>
    </FlipView>
```

## Customization

> The `FlipView` control for desktop targets will present arrow buttons to allow the user to click on the to navigate between the content inside the `FlipView`, if you don't want these arrows to be shown, you can add our `Style` to remove them. Your xaml code will be.

Reviewer: ```suggestion
The `FlipView` control for desktop targets will present arrow buttons to allow the user to click on the to navigate between the content inside the `FlipView`, if you don't want these arrows to be shown, you can use the `FlipViewNoArrowsStyle` as shown in the example below:
```
